src/rabbit_memory_monitor.erl

```diff
@@ -86,7 +86,9 @@ report_ram_duration(Pid, QueueDuration) ->
 stop() ->
     gen_server2:cast(?SERVER, stop).
 
-conserve_resources(Pid, disk, Conserve) ->
+%% Paging should be enabled/disabled only in response to disk resource alarms
```

Comment:
@videlalvaro we can't prevent a node from using disk at all, and even if we could it can't help if another process consumes most space. How exactly messages to flow to the alarmed node doesn't change this. Blocking *all* publishers across all nodes seems excessive to me given that we can't guarantee much as explained above.